index.html

```diff
@@ -97,9 +97,9 @@ <h5 class="cUppercase cWhiteColored cReducedBottomMargin cUnderline cUnderOrange
          <div class="col-sm-12 col-md-4 col-lg-4">
             <div class="cHighlighted cLightGreyBG cHighlightedReducePadding cFixedHeight" style="min-height: 290px;">
                <h4>What's new?</h4>
-               <p>WSO2 Open Banking 1.5.0: Secure regulatory compliance & enable consumer-centric use cases</p>
-               <a href="https://wso2.com/blogs/thesource/wso2-open-banking-1.5-guarantees-global-compliance-and-goes-beyond-to-deliver-new-ROI/" target="_blank" class="cMore cUppercase cLargeText">Read More &gt;</a><br>
-               <a href="https://wso2.com/library/webinars/securing-psd2-compliance-while-boosting-your-open-banking-roi/" target="_blank" class="cMore cUppercase cLargeText">Watch the webinar &gt;</a>
+               <p>WSO2 Open Banking 2.0.0: Secure regulatory compliance & enable consumer-centric use cases</p>
```

Comment:
@isharailanga, is this a general sentence we are going to use for all the releases? If not, shall we mention the changes in the configuration model and revamped UIs specifically? WDYT?

Reply:
Hi Paramee, thanks for bringing this up, this is a general tag line we have used.

However we can update it with the following to bring out the marketing value a little more:
`Standards-compliant support for a microservices architecture and & enable consumer-centric use cases
`
Awaiting your confirmation @lalaji.

Reply:
This will be addressed via https://github.com/wso2/demo-open-banking.github.io/pull/38.